fe/fe-core/src/main/java/org/apache/doris/backup/RestoreJob.java

```diff
@@ -2135,6 +2264,86 @@ public void cancelInternal(boolean isReplay) {
         LOG.info("finished to cancel restore job. is replay: {}. {}", isReplay, this);
     }
 
+    private Status atomicReplaceOlapTables(Database db, boolean isReplay) {
+        assert isAtomicRestore;
+        for (String tableName : jobInfo.backupOlapTableObjects.keySet()) {
+            String originName = jobInfo.getAliasByOriginNameIfSet(tableName);
+            if (Env.isStoredTableNamesLowerCase()) {
+                originName = originName.toLowerCase();
+            }
+            String aliasName = tableAliasWithAtomicRestore(originName);
+
+            if (!db.writeLockIfExist()) {
+                return Status.OK;
```

Comment:
Should we return an error?

Reply:
No, the rest of the code does the same thing, so it's best to be consistent here.